cmd/lncli/main.go

```diff
@@ -2,28 +2,33 @@ package main
 
 import (
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
+	"time"
+
+	"gopkg.in/macaroon-bakery.v1/bakery/checkers"
+	"gopkg.in/macaroon.v1"
```

Comment:
Would like some feedback here: should I submit a PR upstream to fix this issue by helping the original author use glide? If so, should I modify `lnd`'s glide configuration to use my fork unless/until the upstream PR is merged?